learn/on-premise/observability/introduction.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Observability"
+id: ""
+sidebar_label: "Introduction"
+---
+---
+
+Did you ever wonder how a Human body automatically responds to any negative inconvenience?
+
+The body fixes itself with a *Negative Feedback Loop*, which detects the anomalies and performs the required actions to maintain a stable, relatively constant
+ internal environment.
+
+ Example:
+
+![Human Negative Feedback Loop](/learn/assets/wme-setup/wme-observability/negative-feedback-loop-human.png)
+
+*Wouldn't it be great if you had an application with a similar mechanism?* 
+
+Well, we have engineered the WaveMaker Platform to do the same.
+
+![WaveMaker Negative Feedback Loop](/learn/assets/wme-setup/wme-observability/negative-feedback-loop-wavemaker.png)
+
+With the right tools, WaveMaker Platform is smart enough to detect its internal failures and the engineers at WaveMaker are never blinded to anything which
```

Comment:
@sujith-simon the goal is to make this available to our on premise customers also so that they can easily maintain their on premise installation of WaveMaker. We should add that point here.